streamlit_extras/badges/__init__.py

```diff
@@ -63,6 +64,17 @@ def badge(type: str, name: str = None, url: str = None):
                 )
             )
         ) 
+        
+    if type == "creator_support":
```

Comment:
What do you think about calling this type `buymeacoffee`? Instead of `creator_support`? And then using `buymeacoffee_username` instead of `support_page_username`? 

Both for consistency and also because there are other support platforms than buymeacoffee (like https://ko-fi.com/) which could be added here at some point

Reply:
At first, I kept `buymeacoffee` but later I changed it, which was a bad decision though...😅 
I've made the suggested changes and hope it helps 😄